Geometry/CSCGeometry/test/stubs/CSCGeometryAnalyzer.cc

```diff
@@ -42,8 +45,7 @@ void CSCGeometryAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetu
   std::cout << "start " << dashedLine_ << std::endl;
   std::cout << "pi = " << dPi << ", radToDeg = " << radToDeg << std::endl;
 
-  edm::ESHandle<CSCGeometry> pDD;
-  iSetup.get<MuonGeometryRecord>().get(pDD);
+  edm::ESTransientHandle<CSCGeometry> pDD = iSetup.getTransientHandle(ddToken_);
```

Comment:
I think this can be `const` too

Reply:
@tvami Thanks for the suggestions. I made those changes.